### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by aadrian** [huxi/lilith#34]
Hi,

Any chance publishing a new release?

Thank you.

**Comment by huxi** [comment]
Are you having any issues I should be aware of?

The things that changed since the last release are mostly dependency updates.

**Comment by aadrian** [comment]
> Are you having any issues I should be aware of?

In many cases, recent releases/dependencies are a requirement in order to be allowed to deploy a software/tool.

> The things that changed since the last release are mostly dependency updates.

Those would also help :) .

Thank you.

**Comment by huxi** [comment]
**TL;DR:** I'll try to do a release sometime soonish.

My original plan was to have a code base without jdeps warnings on Java 11 for the next release but this hasn't happened so far because of third-party-dependencies.

To get there, I'd at least need to upgrade to Groovy 3.0 (finally available) which would require upgrading to Spock 2.0 (currently at milestone level with "don't use it yet" from the devs) which would require migration to JUnit 5 which would require a replacement of my very own `LoggingTestBase` in `de.huxhorn.sulky.junit`. I have yet to find a reasonable way of implementing that functionality with JUnit 5.

And if all of that was solved, I'd still have zero additional functionality for a huge amount of work.

**This is a work in progress.**
I have a JUnit5 branch but I'm stuck regarding the mentioned issues.

Beside that, the whole Java 8 vs. Java 11/JPMS schism has been seriously bad for my motivation.

Java 8 is Python 2.7. It will stay with us for the next decade.

I saw this coming when the new release cycle was announced (including a new bytecode version for every 6-month-release - which is just insane) and now my premonition is unfolding. Not a single (closed-source) code base in my vicinity has migrated to Java 11, in part despite of a huge effort put into such a migration.

Oracle [just announced](https://www.oracle.com/java/technologies/java-se-support-roadmap.html) extended support for Java 8 until 12-2030. This is more than 4 years longer than extended support for Java 11, the current LTS version.

This essentially means that I (as a library dev) have to stay Java-8-compatible for that long as well, the exact opposite of having fun with new Java features.

Sorry for that rant. I just tried to reflect upon the reasons **why** I have zero motivation for Java development in my spare time anymore. And this is the gist of it. ^^

There are other Java-related sources of frustration but those are the worst.

**Comment by huxi** [comment]
8.3.0 has been released.
